sources/packages/web/src/composables/useFormatters.ts

```diff
@@ -118,12 +118,26 @@ export function useFormatters() {
     };
   };
 
+  /**
+   * format a date to given format
+   * @param date date
+   * @param dateFormat format
+   * @returns extended date format like March, 22 2021
+   */
+  const formatDate = (date: Date | string, dateFormat: string): string => {

```

Comment:
I believe that the idea is concentrate in this _formater_ all the formats needed, so for the case using this method I believe that we already have the method _dateOnlyLongString_.

Reply:
Yeah. But we have 2 date formatters for different formats. This function will simplify that letting us to transform dates flexibly.

Reply:
Using this function the consumer need use this function plus the date format. We need to have the date format centralized, right? 
So why just don't give the format a nick name and consume the method directly?

Reply:
this function is no longer required, removed